cms/djangoapps/contentstore/features/component.py

```diff
@@ -74,8 +74,7 @@ def see_a_problem_component(step, category):
                 'No problem was added to the unit.')
 
     problem_css = 'li.studio-xblock-wrapper div.xblock-student_view'
-    actual_text = world.css_text(problem_css)
-    assert_in(category.upper(), actual_text)
+    assert_true(world.css_contains_text(problem_css, category.upper()))  # pylint: disable=no-value-for-parameter
```

Comment:
@sarina @clytwynec I took another look this morning. The all-caps is how it appears on the page itself. I looked closer at it, creating problems, etc, and this is the behavior of the platform:
![image](https://cloud.githubusercontent.com/assets/2004678/8128920/d674bc3e-10d1-11e5-996e-635795f8115b.png)